docs/dev_guides/manipulators.md

```diff
@@ -0,0 +1,64 @@
+# Manipulators
+
+Manipulators are UI elements attached to napari layer data to aid in positioning and orienting layer data elements 
+being displayed on the canvas. Manipulators can translate and rotate along/around specified axes.
+
+## Coordinate system
+The manipulator coordinate system is defined with an affine transformation relative to the layer data coordinate system.
+The manipulator transformation is defined as a translation and a 3x3 transformation matrix (rotation and scale). The 
+translation is stored in the `manipulator.centroid` property and the transformation is stored in the `manipulator.
```

Comment:
this is consistent with how things are in the code but feels a little unclear generally - how would you feel about a refactor of centroid -> shift moving forwards?

Reply:
I can see how it feels inconsistent. Should we just call it `translation`?